src/main/java/frc/robot/commands/drive/DriveRobot.java

```diff
@@ -17,27 +17,57 @@
 import frc.robot.subsystems.Drive;
 import java.util.function.Supplier;
 
-/** Drives the robot in robot-oriented mode. Default command for {@link Drive} subsystem. */
-public class DriveRobotOriented extends Command {
+/** Drives the robot. */
+public class DriveRobot extends Command {
   private final Drive drive;
   private final Supplier<Double> xSupplier, ySupplier, zSupplier;
+  private final boolean isFieldOriented;
+  private double maxSpeed = RobotContainer.drive.getMaximumSpeed();
+  private double maxAngularSpeed = RobotContainer.drive.getMaximumAngularSpeed();
 
   /**
-   * Drives the robot robot-oriented
+   * Drives the robot.
    *
    * @param xSupplier The joystick value for the Y axis. [-1, 1] left positive.
    * @param ySupplier The joystick value for the Y axis. [-1, 1] back positive.
    * @param zSupplier The joystick value for the Z axis. [-1, 1] counterclockwise positive.
+   * @param isFieldOriented If the robot should drive field oriented or robot oriented.
    */
-  public DriveRobotOriented(
-      Supplier<Double> xSupplier, Supplier<Double> ySupplier, Supplier<Double> zSupplier) {
-    this.drive = RobotContainer.drive;
+  public DriveRobot(
+      Supplier<Double> xSupplier,
+      Supplier<Double> ySupplier,
+      Supplier<Double> zSupplier,
+      boolean isFieldOriented) {
     this.xSupplier = xSupplier;
     this.ySupplier = ySupplier;
     this.zSupplier = zSupplier;
+    this.drive = RobotContainer.drive;
+    this.isFieldOriented = isFieldOriented;
     addRequirements(drive);
   }
 
+  /**
+   * Creates a DriveRobot with
```

Comment:
This line should accurately describe the purpose of the method without relying on @ param calls.